test/test.js

```diff
@@ -347,4 +441,227 @@ describe('rollup-plugin-babel', function() {
 			assert.ok(code.indexOf('console.foobaz') !== -1);
 		});
 	});
+
+	it('can be used as an input plugin while transforming the output', () => {
+		return bundle('samples/basic/main.js', {
+			presets: ['@babel/env'],
+			transformGenerated: true,
+		}).then(({ code }) => {
+			assert.ok(code.indexOf('const') === -1, code);
+		});
+	});
+
+	describe('when used as an output plugin', () => {
```

Comment:
Instead of nesting this describe block I think we could just make it top level one (for better readability)